### PR TITLE
Wave2 1051 styling autocomplete

### PR DESCRIPTION
**Pull request by esoltys**
Autocomplete styling on top of latest WAVE2-1028 build.

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_all_programs/js/autocomplete.js [hunk 1]
```diff
@@ -0,0 +1,330 @@
+(function ($) {
```
Comment (gsowrey) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_all_programs/js/autocomplete.js]: If this is a global autocomplete function, should it not be ... global?

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_all_programs/js/ucgy_fgs_all_programs.js [hunk 2]
```diff
@@ -238,11 +238,13 @@
 
       function updateProgramCount() {
         var initialCount = 0 | $('#view-count .count').text();
+        console.log(initialCount);
         // Get a count of visible program degrees
-        var currentCount = $(".programs > .row > .container > .row > .program").not(".minimized").size();
+        var currentCount = $(".programs > .row > .container > .row > .program").not(".minimized, .related-programs-list").size();
+        console.log(currentCount);
```
Comment (gsowrey) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_all_programs/js/ucgy_fgs_all_programs.js]: BAH! please remove before going live!

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_all_programs/templates/views-view--ucgy-fgs-all-programs-solr.tpl.php [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 <?php
 drupal_add_js(drupal_get_path('module', 'ucgy_fgs_all_programs') . '/js/ucgy_fgs_all_programs.js');
+drupal_add_js(drupal_get_path('module', 'ucgy_fgs_all_programs') . '/js/autocomplete.js');
```
Comment (gsowrey) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_all_programs/templates/views-view--ucgy-fgs-all-programs-solr.tpl.php]: Again, if this is global, shouldn't we have this as a global inclusion?

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_all_programs/templates/views-view--ucgy-fgs-all-programs.tpl.php [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 <?php
 drupal_add_js(drupal_get_path('module', 'ucgy_fgs_all_programs') . '/js/ucgy_fgs_all_programs.js');
+drupal_add_js(drupal_get_path('module', 'ucgy_fgs_all_programs') . '/js/autocomplete.js');
```
Comment (gsowrey) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_all_programs/templates/views-view--ucgy-fgs-all-programs.tpl.php]: Global functionality, global use?

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_all_programs/templates/views-view--ucgy-fgs-all-programs.tpl.php [hunk 5]
```diff
@@ -2,6 +2,7 @@
 
 <?php
 drupal_add_js(drupal_get_path('module', 'ucgy_fgs_all_programs') . '/js/ucgy_fgs_all_programs.js');
```
Comment (gsowrey) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_all_programs/templates/views-view--ucgy-fgs-all-programs.tpl.php]: Also, we shouldn't be adding JS at the _template_ level -- this should only be added in the .module

**Comment by dpfitzsi** [comment]
New PR opened to fix code that got added incorrectly.

**Pull request closed without merging**